R/deltaccd.R

```diff
@@ -152,33 +152,46 @@ calcCCD = function(
     stop('refCor must be a correlation matrix, with identical rownames and colnames.')}
 
   geneNames = rownames(refCor)[rownames(refCor) %in% rownames(emat)]
-  refNow = refCor[geneNames, geneNames]
-  if (length(geneNames) < 2) {
-    stop('Fewer than two genes in the reference are in the expression matrix.')
-  } else if (length(geneNames) < nrow(refCor)) {
-    warning(sprintf('%d gene(s) in reference is/are not in the expression matrix.',
-                    nrow(refCor) - length(geneNames)))}
+  if (length(geneNames) < nrow(refCor)) {
```

Comment:
Let's put this in a separate function, so we can minimize redundancy in calcCCD and calcDeltaCCD. Also we should print out the actual genes, not just how many there are.